tests/tacacs/test_ro_user.py

```diff
@@ -77,6 +78,22 @@ def wait_for_tacacs(localhost, remote_ip, username, password):
                 current_attempt += 1
 
 
+def ssh_remote_run_with_ipv6(localhost, dutip, ptfhost, user, password, command):
+    retry_count = RETRY_COUNT
+    while retry_count > 0:
+        res = ssh_remote_run(localhost, dutip, user,
+                             password, command)
+
+        # TACACS server randomly crash after receive authorization request from IPV6
```

Comment:
>crash [](http://example.com/codeflow?start=33&length=5)

Wondering what happens after TACACS crash? did it auto restart?

Reply:
It not auto restarts because it not a systemd service. and not found any error message in syslog and tacplus service log.